e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -125,6 +126,21 @@ Scenario('HMCTS admin enters hearing details and submits', async (I, caseViewPag
   I.seeAnswerInTab(9, 'Hearing 2', 'Judge or magistrate\'s last name', hearingDetails[1].lastName);
 });
 
+Scenario('HMCTS admin creates C21 order for the case', async (I, caseViewPage, createC21OrderEventPage) => {
+  await caseViewPage.goToNewActions(config.administrationActions.createC21Order);
+  await createC21OrderEventPage.enterOrder();
+  await I.click('Continue');
```

Comment:
await is not needed

Reply:
I had a bug where the legal advisor was being entered in the judge last name field instead, mixed with the judge's last name, causing the test to fail - using await fixed that at the time. can remove and retest

Reply:
await on the I click though?